core/domain/suggestion_registry.py

```diff
@@ -1095,3 +1095,95 @@ def reject(self, reviewer_id, rejection_message):
         SuggestionTranslateContent),
     suggestion_models.SUGGESTION_TYPE_ADD_QUESTION: SuggestionAddQuestion
 }
+
+
+class ReviewerAndSuggestionCounts(python_utils.OBJECT):
+    """Domain object for the ReviewerAndSuggestionCountsModel.
+
+    Attributes:
+        translation_reviewer_counts_per_lang: dict. A dictionary that contains
+            the total number of translation reviewers for each language that
+            translation suggestions are offered in.
+        translation_suggestion_counts_per_lang: dict. A dictionary that
+            contains the total total number of translation suggestions that are
+            currently in review per language.
+        question_reviewer_count: int. The total number of reviewers who have
+            permission to review question suggestions.
+        question_suggestion_count: int. The total number of question
+            suggestions that are currently in review.
+    """
+
+    def __init__(
+            self, translation_reviewer_counts_per_lang,
```

Comment:
Perhaps: translation_reviewer_counts_by_lang_code, translation_suggestion_counts_by_lang_code (lang_code is for clarity, and I think we also typically use `_by_` for this sort of thing)